crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -102,7 +102,7 @@ reveal_type(C.pure_instance_variable)  # revealed: str
 # and pyright allow this.
 C.pure_instance_variable = "overwritten on class"
 
-# error: [invalid-assignment] "Object of type `Literal[1]` is not assignable to `str`"
+# error: [invalid-assignment] "Object of type `Literal[1]` is not assignable to attribute of type `str`"
```

Comment:
@AlexWaygood I intentionally changed "public type" to just "type" here, as I'm not sure if "public type" is a term that we want to teach to users?

Reply:
yeah, you're probably right -- it's probably unnecessary jargon